src/__tests__/__snapshots__/typescript.js.snap

```diff
@@ -1,7 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Typescript expected failures 1`] = `
-"test/should-fail.test.tsx(10,3): error TS2345: Argument of type '{ fillRule: \\"cat\\"; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}, object>'.
+"test/glamorous.test.tsx(175,31): error TS2322: Type 'HTMLDivElement | null' is not assignable to type 'HTMLDivElement'.
```

Comment:
It's not no.  I'm unsure what caused that, none of the dependencies look like they should have impacted this.

It doesn't appear to have occurred on https://github.com/paypal/glamorous/pull/246 though, if it happens again I'll investigate further.

Also noticed that this test completely fails on windows due to inconstancies with spawnSync :(.  I'll hopefully land something in the coming days to resolve this.